src/scripts/generateTranslations.js

```diff
@@ -0,0 +1,71 @@
+import "dotenv/config";
+import { createClient } from "@sanity/client";
+import * as fs from "node:fs";
+
+export const client = createClient({
+  projectId: process.env.SANITY_STUDIO_PROJECT_ID,
+  dataset: process.env.SANITY_STUDIO_DATASET,
+  useCdn: false, // set to `false` to bypass the edge cache
+  apiVersion: "2023-05-03", // use current date (YYYY-MM-DD) to target the latest API version
+});
+
+const visualEditingEnabled =
+  process.env.SANITY_VISUAL_EDITING_ENABLED === "true";
+const token = process.env.SANITY_API_READ_TOKEN;
+
+export async function loadQuery({ query, params }) {
```

Comment:
We shouldn't export the method since it is only used in this file if I'm correct. This method name confused me a bit since there is a `@api/sanity` method with the same name.